build/DeployTasks/UploadArtifactsTask.cs

```diff
@@ -19,6 +19,9 @@ public override async Task RunAsync(BuildContext context)
         await context.GitHubActions().Commands.UploadArtifact(new DirectoryPath(System.IO.Path.Combine(context.BuildOutput, "Tests", "Tools", "Release")), $"tests-tools-{os}");
         await context.GitHubActions().Commands.UploadArtifact(new DirectoryPath(System.IO.Path.Combine(context.BuildOutput, "Tests", "DesktopGL", "Release")), $"tests-desktopgl-{os}");
         if (context.IsRunningOnWindows())
+        {
             await context.GitHubActions().Commands.UploadArtifact(new DirectoryPath(System.IO.Path.Combine(context.BuildOutput, "Tests", "WindowsDX", "Release")), $"tests-windowsdx-{os}");
+            await context.GitHubActions().Commands.UploadArtifact(new DirectoryPath(System.IO.Path.Combine(context.BuildOutput, "MonoGame.Templates.VSExtension", "net472")), "MonoGame.Templates.VSExtension.vsix");
```

Comment:
I am not sure how vsix files work, but this will zip up the entire net472 directory and upload it as artifact named MonoGame.Templates.VSExtension.vsix.

Did you by any chance want to specify the file path instead of the entire directory or was this intentional?

Reply:
@harry-cpp I wanted just the file itself, if possible. Vsixs are already a type of zip file. What changes are required?

Reply:
@harry-cpp would this work:
```
var vsixFilePath = System.IO.Path.Combine(context.BuildOutput, "MonoGame.Templates.VSExtension", "net472", "MonoGame.Templates.VSExtension.vsix");
await context.GitHubActions().Commands.UploadArtifact(new FilePath(vsixFilePath), "MonoGame.Templates.VSExtension.vsix");
```

Reply:
It looks like it did just zip up just that vsix and nothing else.